docs/pages/project/contributing/contributing-docs.md

```diff
@@ -401,5 +383,38 @@ The assign tag is used to create a new variable. It is written in the following
 {% capture code_content %}{{ "{% assign variable1 = true " }}%}{% endcapture %}
 {% include code.html code=code_content %}
 
+### Alert
+
+The `alert.html` file is used to format and display notes, warnings, and other informational callouts in Meshery. 
+It provides a consistent and visually appealing way to draw the user's attention to important information.
+
+To use the `alert.html` feature, include the following code snippet in your documentation:
+``` 
+    &lt;div class="alert alert-{{ include.type }}"&gt;
+    &lt;h4 class="alert-heading"&gt;{% if include.title %}{{ include.title }}{% else %}{{ include.type }}{% endif %}&lt;/h4&gt;
+    {{ include.content }}
+    &lt;/div&gt;
+```
+    
+This code defines an HTML structure for creating alerts. It accepts three parameters:
+```
+
+        {{ include.type }}: Specifies the type of alert, which can be "info," "warning," "success," or any other relevant value. This is used to style the alert
+        {{ include.title }} : Sets the title of the alert. If not provided, it will default to the value of <code>{{ include.type }}</code>
+        {{ include.content }}: Contains the content of the alert, such as the message or information you want to convey.
+ ``` 
```

Comment:
```suggestion

{{ include.type }}: Specifies the type of alert, which can be "info," "warning," "success," or any other relevant value. This is used to style the alert
 {{ include.title }} : Sets the title of the alert. If not provided, it will default to the value of <code>{{ include.type }}</code>
 {{ include.content }}: Contains the content of the alert, such as the message or information you want to convey.

```